code/modules/mob/living/carbon/human/human.dm

```diff
@@ -3,7 +3,8 @@
 	real_name = "Unknown"
 	voice_name = "Unknown"
 	icon = 'icons/mob/human.dmi'
-	icon_state = "caucasian1_m_s"
+	icon_state = "caucasian1_m_s"
+	var/acedia = 0

```

Comment:
Add a function that checks for objective type in the objective list instead of adding this please.